.gitattributes

```diff
@@ -9,3 +9,4 @@ src/test/resources/large/funcotator/funcotator_dataSources/simple_uniprot/hg38 -
 src/test/resources/large/** filter=lfs diff=lfs merge=lfs -text
 src/main/resources/large/** filter=lfs diff=lfs merge=lfs -text
 *.psd filter=lfs diff=lfs merge=lfs -text
+src/test/resources/1kgp.ped filter=lfs diff=lfs merge=lfs -text
```

Comment:
We don't add small files like this test one to lfs usually.   Only large files that don't work well within git should use lfs (>1 mb is usually where we consider the cutoff but it's fuzzy, if you're adding 100 750k files then they should probably go in lfs). 

By convention we make sure we only ever use lfs in the resources/large directories so it's clear when there are problems.

Reply:
Typically you'd put it in a resource folder who's structure matches the package structure of the class under test.